fusedrug/data/tokenizer/modulartokenizer/multi_tokenizer.py

```diff
@@ -333,12 +412,33 @@ def fix_json_paths(
                 )
             return loaded_conf
 
-        loaded_conf = OmegaConf.load(os.path.join(path, "config.yaml"))
-        loaded_conf_fixed = fix_json_paths(loaded_conf, path)
-        return ModularTokenizer(tokenizers_info=loaded_conf_fixed, load_adjusted_jsons=True)
+        try:
+            loaded_conf: omegaconf.dictconfig.DictConfig = OmegaConf.load(os.path.join(path, "config.yaml"))
+        except:
+            raise Exception(f"couldn't load config.yaml from {path}")
```

Comment:
please also print the exception (e.g. traceback.print_exc())

Reply:
Done